go-controller/pkg/cni/helper_linux.go

```diff
@@ -339,12 +348,21 @@ func (pr *PodRequest) ConfigureInterface(namespace string, podName string, ifInf
 	}
 
 	err = netns.Do(func(hostNS ns.NetNS) error {
+		// deny IPv6 neighbor solicitations
 		if _, err := os.Stat("/proc/sys/net/ipv6/conf/all/dad_transmits"); !os.IsNotExist(err) {
 			err = setSysctl("/proc/sys/net/ipv6/conf/all/dad_transmits", 0)
 			if err != nil {
 				klog.Warningf("Failed to disable IPv6 DAD: %q", err)
 			}
 		}
+		// generate address based on EUI64
+		if _, err := os.Stat("/proc/sys/net/ipv6/conf/all/addr_gen_mode"); !os.IsNotExist(err) {
```

Comment:
hm... this one and the `dad_transmits` one should probably both be only for `conf/eth0` not `conf/all`. We should not be overriding any settings on Multus interfaces, right?

Reply:
good catch

Reply:
done